pinot-core/src/main/java/org/apache/pinot/core/query/pruner/ColumnValueSegmentPruner.java

```diff
@@ -59,8 +62,11 @@
 @SuppressWarnings({"rawtypes", "unchecked"})
 public class ColumnValueSegmentPruner implements SegmentPruner {
 
+  private int _inPredicateThreshold;
+
   @Override
   public void init(PinotConfiguration config) {
+    _inPredicateThreshold = config.getProperty(Server.CONFIG_THRESHOLD_FOR_IN_PREDICATE, Server.DEFAULT_VALUE_FOR_IN_PREDICATE_THRESHOLD);
```

Comment:
Should we exclude the prefix `pinot.server.query.executor.pruner.columnvaluesegmentpruner` from the config key?

Reply:
I think it might make the name look cleaner. However, keeping the dot notation based name makes it consistent with all the config keys we have today. Also the name makes it self explanatory that it is for pruner on server query execution

Reply:
sounds okay to me to keep the prefix